types/src/front/lib/assistant.ts

```diff
@@ -87,6 +87,7 @@ export const GPT_4_TURBO_MODEL_ID = "gpt-4-turbo" as const;
 export const GPT_4O_LEGACY_MODEL_ID = "gpt-4o" as const;
 export const GPT_4O_MODEL_ID = "gpt-4o-2024-08-06" as const;
 export const GPT_4O_MINI_MODEL_ID = "gpt-4o-mini" as const;
+export const O1_MODEL_ID = "o1-preview" as const;
```

Comment:
nit: should we call this `O1_PREVIEW_MODEL_ID` instead ? And maybe also support `O1_MINI_MODEL_ID` ?

Reply:
Done for the rename, and I also added a global agent for mini, with a different feature flag so we can activate one or the other or both